go-controller/pkg/ovn/namespace.go

```diff
@@ -271,6 +277,20 @@ func (oc *Controller) updateNamespace(old, newer *kapi.Namespace) {
 			}
 		}
 	}
+	annotation = newer.Annotations[aclLoggingAnnotation]
+	oldAnnotation = old.Annotations[aclLoggingAnnotation]
+	// support for ACL logging update, if new annotation is empty, make sure we propagate new setting
+	if annotation != oldAnnotation && (annotation == "" || oc.aclLoggingCanEnable(annotation, nsInfo)) {
+		// deny rules are all one per namespace
+		for _, np := range nsInfo.networkPolicies {
+			if err := oc.setACLDenyLogging(np, annotation); err != nil {
```

Comment:
you are passing `annotation` here to setACLDenyLogging, i.e., the entire JSON blob however in the function `setACLDenyLogging` it is used as a string "aclLoggingLevel.Deny"? can you please fix?